Automattic/jetpack
# jetpack: Move postcss.config.js into tools and update comment

#### Changes proposed in this Pull Request:

We're always specifying it by name, although it seems `postcss-loader`
will search for it there (even when we don't want it to).

Also, after a lot of git archaeology, I figured out what was going on
with the `preserve: false` thing. It was never actually a "workaround"
as was thought, calypso-build had been setting exactly the same thing in
v4.2.1. We might still want to remove it someday to more properly make
use of CSS variables, but first we'll need to work out how to make sure
all those variables get set in the browser without copying them in every
stylesheet.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Is the build happy?
* There should not be any changes in the build artifacts. Is that the case?
